.github/workflows/release.yaml

```diff
@@ -122,3 +122,10 @@ jobs:
     uses: celo-org/developer-tooling/.github/workflows/open-docs-pr.yml@master
     with:
       commit: ${{ github.sha }}
+
+  upload-executables:
+    needs: release
+    if: ${{ contains(fromJson(needs.release.outputs.publishedPackages).*.name, '@celo/celocli') }}
```

Comment:
does it limit to only production releases elsewhere? or can we do that here?

Reply:
done in c16d7bfaa